src/measure_theory/function/ae_eq_of_integral.lean

```diff
@@ -471,4 +471,52 @@ end
 
 end ae_eq_of_forall_set_integral_eq
 
+section lintegral
+
+-- move the next two lemmas once I figure out if `eventually_ne_of_lt` exists already
+-- if not, it might be useful to make some API for eventually lt
```

Comment:
AFAIR, we don't have these lemmas. Feel free to add (untested; I omit implicit args):
```lean
lemma filter.eventually.ne_of_lt (h : ∀ᶠ x in l, f x < g x) : ∀ᶠ x in l, f x ≠ g x :=
h.mono (λ x hx, hx.ne)

lemma filter.eventually.ne_top_of_lt (h : ∀ᶠ x in l, f x < g x) : ∀ᶠ x in l, f x ≠ ∞ :=
h.mono (λ x hx, hx.ne_top)

lemma filter.eventually.lt_top_of_ne (h : ∀ᶠ x in l, f x ≠ ∞) : ∀ᶠ x in l, f x < ∞ :=
h.mono (λ x hx, hx.lt_top)
```

Reply:
Thanks!